visual-testing-app/src/components/info-detail-page/info-detail-page.visualroute.tsx

```diff
@@ -0,0 +1,59 @@
+import Text from '@commercetools-uikit/text';
```

Comment:
GitHub doesn't have syntax highlighting on this file but I guess that's unrelated to the file itself.